beanie/odm/utils/pydantic.py

```diff
@@ -1,9 +1,22 @@
 from typing import Any, Type
 
 import pydantic
+from packaging.specifiers import SpecifierSet
```

Comment:
You'll need to switch back to the `pydantic.VERSION` manual parsing as `packacing` is a 3rd party dependency, and currently not included in the requirements.

Reply:
I agree here, lets not introduce `packaging` as a dependency.